cg/cli/workflow/balsamic/store.py

```diff
@@ -23,77 +25,105 @@
 def store(context):
     """Store results from MIP in housekeeper."""
     context.obj["db"] = Store(context.obj["database"])
-    context.obj["tb_api"] = tb.TrailblazerAPI(context.obj)
     context.obj["hk_api"] = hk.HousekeeperAPI(context.obj)
+    context.obj["analysis_api"] = AnalysisAPI(
```

Comment:
In this not instanciated earlier in the balsamic base command?

Reply:
utilised now